lib/sisimai/smtp/error.rb

```diff
@@ -89,11 +89,13 @@ def soft_or_hard(argv1 = '', argv2 = '')
             getchecked = Sisimai::SMTP::Error.is_permanent(argv2)
 
             if getchecked.nil?
-              # The value is not defined (returned undef)
               softorhard = ''
             else
-              # The value is 0 or 1
-              softorhard = getchecked == 1 ? 'hard' : 'soft'
+              softorhard = if getchecked
+                'hard'
+              else
+                'soft'
```

Comment:
__[rubocop]__
* Layout/IndentationWidth: Use 2 (not -11) spaces for indentation. [:ref](https://github.com/bbatsov/ruby-style-guide#spaces-indentation)

<!-- Comment by [SideCI](https://sideci.com) -->